api/MessageEvent.json

```diff
@@ -94,7 +94,7 @@
               "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": "37"
```

Comment:
This is an outlier change in this PR, where the original data was "37" instead of "1". But it seems correct to me, I don't believe this was introduced in WebView 37 specifically.